### PR TITLE
Replace buffer with atob()

### PR DESCRIPTION
**Pull request by isAdrisal**
Removes the node import by replacing buffer with atob().

### CODE REVIEW

File: wrangler.toml [hunk 1]
```diff
@@ -1,4 +1,2 @@
 name = "unifi-cloudflare-ddns"
 main = "src/index.js"
-compatibility_date = "2023-04-23"
```
Comment (willswire) [inline wrangler.toml:3]: Let's keep the `compatibility_date` property in the `wrangler.toml`

File: package.json
Comment (willswire) [inline package.json:1]: Let's remove the entire `package.json`

**Review by willswire — CHANGES_REQUESTED** [review]
Great contribution! Removing the need to leverage NodeJS for this worker is helpful. Thanks!

File: wrangler.toml (hunk 1 above)
Comment (isAdrisal) [reply wrangler.toml:3]: Ah that's my bad, I thought this was used for node compat. Reverted :)

File: package.json
Comment (isAdrisal) [reply package.json:1]: Done!

**Comment by isAdrisal** [comment]
> Great contribution! Removing the need to leverage NodeJS for this worker is helpful. Thanks!

No worries, thanks for putting the repo together 🙏

**Pull request merged**